HyperVExtension/src/HyperVExtension/Providers/HyperVProvider.cs

```diff
@@ -112,4 +151,32 @@ public ComputeSystemAdaptiveCardResult CreateAdaptiveCardSessionForComputeSystem
             return null;
         }
     }
+
+    private string SetupHyperVPreReqErrorText()
+    {
+        var availabilityKind = WmiUtility.GetHyperVFeatureAvailability();
+        if (availabilityKind == FeatureAvailabilityKind.Disabled)

```

Comment:
Would a switch statement be better here?

Reply:
sure I can update it if its easier on the eyes